proxy/src/http/user.rs

```diff
@@ -346,23 +352,33 @@ mod test {
             .await
             .unwrap();
 
-        let api = super::filters(registry, store, subscriptions);
+        let api = super::filters(Arc::clone(&registry), store, subscriptions);
```

Comment:
Just wondering for my own knowledge - why should this be `clone`d here?

Reply:
I introduced a new place were `registry` is being used [L.362](https://github.com/radicle-dev/radicle-upstream/pull/669/files/ffe70a3beb5dfc071244b406e244c9cd85452033#diff-adc9819afa99f02dc58458363a1e459bR362), so to use `registry` there, I need to clone it here.